workflows/data_pipelines/colter/colter_dag.py

```diff
@@ -0,0 +1,118 @@
+from datetime import timedelta
+
+from airflow.decorators import dag, task
+from airflow.utils.dates import days_ago
+from airflow.datasets import Dataset
+
+from dag_datalake_sirene.config import EMAIL_LIST
+from dag_datalake_sirene.helpers import Notification
+from dag_datalake_sirene.workflows.data_pipelines.colter.colter_config import (
+    COLTER_CONFIG,
+    ELUS_CONFIG,
+)
+from dag_datalake_sirene.workflows.data_pipelines.colter.colter_processor import (
+    ColterProcessor,
+    ElusProcessor,
+)
+
+default_args = {
+    "depends_on_past": False,
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "email": EMAIL_LIST,
+    "retries": 1,
+}
+
+dataset_colter = Dataset(COLTER_CONFIG.name)
+
+
+@dag(
+    tags=["collectivités", "communes", "régions", "départements"],
+    default_args=default_args,
+    schedule_interval="0 16 * * *",
+    start_date=days_ago(8),
+    dagrun_timeout=timedelta(minutes=60),
+    on_failure_callback=Notification.send_notification_tchap,
+    on_success_callback=Notification.send_notification_tchap,
+    params={},
+    catchup=False,
+)
+def data_processing_collectivite_territoriale():
+    colter_processor = ColterProcessor()
+
+    @task.bash
+    def clean_previous_outputs():
+        return (
+            f"rm -rf {COLTER_CONFIG.tmp_folder} && mkdir -p {COLTER_CONFIG.tmp_folder}"
+        )
+
+    @task()
+    def preprocess_data():
+        return colter_processor.preprocess_data()
+
+    @task
+    def save_date_last_modified():
+        return colter_processor.save_date_last_modified()
+
+    @task
+    def send_file_to_minio():
+        return colter_processor.send_file_to_minio()
+
+    @task
+    def compare_files_minio(outlets=[dataset_colter]):
+        return colter_processor.compare_files_minio()
+
+    (
+        clean_previous_outputs()
+        >> preprocess_data()
+        >> save_date_last_modified()
+        >> send_file_to_minio()
+        >> compare_files_minio()
+    )  # type: ignore
```

Comment:
Meaning?

Reply:
To ignore Pylance "expression value is unused" warning.

Reply:
What's unused exactly?

Reply:
This is Pylance complaining 😅 
<img width="497" alt="image" src="https://github.com/user-attachments/assets/e1af25ce-37e1-4f2e-ba8d-fb26182c7570" />
It expects this:
<img width="275" alt="image" src="https://github.com/user-attachments/assets/ae6664e7-cea0-4cff-814c-0e572411586a" />
But I just went with adding the ignore instead haha

Reply:
That's strange. Why is the error not raised for the other dags ?

Reply:
That's just a linting warning, not a real error, it was also raised but I ignored it until then 😅
Do you also use Pylance with VSCode?

Reply:
Nope! I stick to the linting in precommit to make sure it's common for all code owners.

Reply:
Sorry I'm insisting because I don't really like this type or styling comments in code in general.
I used it for line breaks for Black in imports and I think it disturbs the reading. That's why I asked.